Library/Client.ts

```diff
@@ -131,17 +132,22 @@ class Client {
         this.track(data);
     }
 
-    public trackRequestSync(request: http.ServerRequest, response: http.ServerResponse, ellapsedMilliseconds?: number, properties?: { [key: string]: string; }, error?: any) {
-        RequestTracking.trackRequestSync(this, request, response, ellapsedMilliseconds, properties, error);
+    public trackServerRequestSync(request: http.ServerRequest, response: http.ServerResponse, ellapsedMilliseconds?: number, properties?: { [key: string]: string; }, error?: any) {
+        ServerRequestTracking.trackRequestSync(this, request, response, ellapsedMilliseconds, properties, error);
     }
 
-    public trackRequest(request: http.ServerRequest, response: http.ServerResponse, properties?: { [key: string]: string; }) {
-        RequestTracking.trackRequest(this, request, response, properties);
+    public trackServerRequest(request: http.ServerRequest, response: http.ServerResponse, properties?: { [key: string]: string; }) {
+        ServerRequestTracking.trackRequest(this, request, response, properties);
+    }
+
+    public trackClientRequest(requestOptions: any, request: http.ClientRequest, properties?: { [key: string]: string; }) {
+        ClientRequestTracking.trackRequest(this, requestOptions, request, properties);
     }
 
     public trackDependency(
         name: string,
         commandName: string,
+        target: string,
```

Comment:
I don't think our bond currently asks for this field to be required. It might not be so bad to move this to the end and mark optional. Anyone have strong thoughts on that?